package.json

```diff
@@ -25,7 +25,9 @@
     "lint": "eslint .",
     "prettier": "prettier --write",
     "detect-port": "cross-env ./node_modules/detect-port-alt/bin/detect-port 3000",
-    "precommit": "yarn run flow && lint-staged"
+    "precommit": "yarn run flow && lint-staged",
+    "build-storybook": "build-storybook -c .storybook -o storybook-static",
```

Comment:
I copied this bit from @superhawk610's PR. Because it's character-for-character, though, there shouldn't be a conflict :)